app/models/concerns/course/discussion/post/ordering_concern.rb

```diff
@@ -41,18 +41,10 @@ def sort(post_id)
     end
   end
 
-  module ScopeMethods
-    # Returns a set of recursive arrays indicating the parent-child relationships of posts.
-    #
-    # @return [Enumerable]
-    def ordered_topologically
-      PostSort.new(self)
-    end
-  end
-
-  included do
-    scope :ordered_topologically, (lambda do
-      ScopeMethods.instance_method(:ordered_topologically).bind(current_scope).call
-    end)
+  # Returns a set of recursive arrays indicating the parent-child relationships of posts.
+  #
+  # @return [Enumerable]
+  def ordered_topologically
```

Comment:
So it servers same functionality as the original scope which you removed right ?